client/src/components/header/Header.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { Rules } from "../Rules/Rules";
+import { Rules } from "../rules/Rules";
```

Comment:
It looks like the Rules folder has been set back to rules. Could you re-capitalise it please?